doc/source/developing/building_the_docs.rst

```diff
@@ -90,7 +90,7 @@ the top level of a local copy, run
 
 .. code-block:: bash
 
-   $ python -m pip install -e . -r requirements/docs.txt
+   $ python -m pip install -e . --group docs
```

Comment:
the `groups` functionality necessitates, `uv`, ya? this command fails as is with `no such option: --group`. i think you probably mean `uv run pip install -e . --group docs` ? 

and would using uv here change the rest of the docs building? you'd at least need to activate the environment before running the build command? 

related: removing the `requirements/docs.txt` would likely break whatever script builds and publishes the sphinx doc. I know that server is down... but I'm a little hesitant to make this change while the docs are down, might delay the process of getting it up and running again? how would you feel about keeping `requirements/docs.txt` separate from the new uv dependency groups?

Reply:
though if you're still leaning towards keeping the requirements file, that's fine with me, just let me know !

Reply:
>  but you do need version 25.1 

ah! ok, i was probably on an old pip.

> I suspect your (otherwise legitimate) concern could be inflated by your earlier confusion over dependency group support ?

yes, exactly :) I was worried we were getting tied into uv for our docs pipeline (which I might actually get behind, but thought it should be separate from this PR). But since it's just a pip version issue, let's remove the docs requirements.